lib/src/widgets/chat/ds_location_message_bubble.widget.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:blip_ds/blip_ds.dart';
```

Comment:
Use relative imports, but not the ```import '../../../blip_ds.dart'``` one. In this case, you it's better to add every import individually:

```import '../../enums/ds_align.enum.dart';```
```import '../../models/ds_message_bubble_style.model.dart';```
```import '../../themes/colors/ds_colors.theme.dart';```
```import '../animations/ds_spinner_loading.widget.dart';```
```import '../texts/ds_body_text.widget.dart';```
```import '../utils/ds_cached_network_image_view.widget.dart';```
```import 'ds_message_bubble.widget.dart';```